### PR TITLE
feat: Kiva Classic typography and form element updates for /checkout

### PR DESCRIPTION
**Pull request by ryan-ludwig**
https://www.test.kiva.org/checkout

Note, this PR does not include the thanks page, only checkout.

### CODE REVIEW

File: src/components/Checkout/DonationNudge/DonationNudgeBoxes.vue [hunk 1]
```diff
@@ -406,6 +410,8 @@ export default {
 						.nudge-box-input {
 							text-align: center;
 							margin: 0 auto;
+							padding: 0.5rem;
```
Comment (ryan-ludwig) [inline src/components/Checkout/DonationNudge/DonationNudgeBoxes.vue:413]: Normally I'd be removing these kind of color values, not adding them. But once I started going through this particular dialog it seemed smarter to fix a couple things and then make a ticket for a larger pass later.

**Review by mcstover — APPROVED** [review]
I have a feeling this is moving us pretty far into KC but I'm excited to see it!

File: src/components/Checkout/LoanReservation.vue [hunk 2]
```diff
@@ -1,56 +1,54 @@
...
+			</p>
+		</kv-lightbox>
 	</div>
 </template>
 
 <script>
 import { differenceInMinutes, differenceInSeconds } from 'date-fns';
-import KvButton from '@/components/Kv/KvButton';
-import KvLightbox from '@/components/Kv/KvLightbox';
+import KvLightbox from '~/@kiva/kv-components/vue/KvLightbox';
```
Comment (emuvente) [inline src/components/Checkout/LoanReservation.vue:48]: Seeing this reminds me that we should update the kv-components KvLightbox to also have this change: https://github.com/kiva/ui/commit/9263531b3a58636d675fc0c0060c5abadfb756ee

File: src/components/Checkout/OrderTotals.vue [hunk 3]
```diff
@@ -4,7 +4,7 @@
 			<strong>Order Total: <span class="total-value">{{ itemTotal }}</span></strong>
 		</div>
 
-		<div v-if="showKivaCredit" class="kiva-credit" data-test="kiva-credit">
+		<div v-if="showKivaCredit || true" class="kiva-credit tw-font-medium tw-mb-2" data-test="kiva-credit">
```
Comment (emuvente) [inline src/components/Checkout/OrderTotals.vue:7]: `|| true` for debugging?

Comment (ryan-ludwig) [reply src/components/Checkout/OrderTotals.vue:7]: Ack! Thank you!

File: src/components/Checkout/CheckoutDropInPaymentWrapper.vue [hunk 4]
```diff
@@ -80,9 +80,9 @@
 				<kv-button
 					value="submit"
 					id="dropin-submit"
-					class="button"
+					class="tw-mb-2"
 					:disabled="!enableCheckoutButton"
```
[the remark this answers is not in the document]
Comment (ryan-ludwig) [reply src/components/Checkout/CheckoutDropInPaymentWrapper.vue:84]: Good catch! No, it's now `:state="disabled | loading"`

[the remark this answers is not in the document]
Comment (ryan-ludwig) [reply src/components/Checkout/CheckoutDropInPaymentWrapper.vue:84]: I'll need to check the other buttons for this

**Review by emuvente — APPROVED** [review]
Nice!

**Pull request merged**